### DISCUSSION TITLE
3.5.2 has page boundary delay problem while scrolling down

### DISCUSSION THREAD

**Discussion by balmydrizzle** [sumatrapdfreader/sumatrapdf#4672] [General]
This problem never happen before 3.5.2 and it has same problem exactly like Foxit PDF after v9 or v10, i.e. whenever browsing down across page boundary, there would be an unpleasant lag(or so delay), it make me feel uncomfortable while reading through pdf file. Can this be fixed? I ask this because I complaint mant times but Foxit PDF just can't fix ir. Sometimes programming in too much modularized or too much structured (due to consider too many functions) might cause this problem.

**Comment by GitHubRulesOK** [comment]
The pages in a PDF are stored as random objects NOT always sequential, thus as more complex pages need loading there is a delay while the pages are found from the index in the trailer, decrypted, decoded, decompressed, and converted to pixels for send to the screen.

Best speed between pages is gained by less pages, using non encrypted files that are not high-density contents and are NOT compressed.
